super_gui/src/components/DoorManUI.tsx

```diff
@@ -0,0 +1,166 @@
+import { useState } from "react";
+
+// Interface for package lockers
+interface PackageLocker {
+  id: number;
+  lockerNumber: number;
+  status: string;
+  apartmentNumber?: string;
+}
+
+const DoorManUI = () => {
+  const [apartmentNumber, setApartmentNumber] = useState("");
+  const [selectedLocker, setSelectedLocker] = useState("");
+
+  // Mock data for lockers
+  const [lockers, setLockers] = useState<PackageLocker[]>([
+    { id: 1, lockerNumber: 1, status: "Apartment #4", apartmentNumber: "4" },
+    { id: 2, lockerNumber: 2, status: "Empty" },
+    { id: 3, lockerNumber: 3, status: "Empty" },
+    { id: 4, lockerNumber: 4, status: "Empty" },
+    { id: 5, lockerNumber: 5, status: "Empty" },
+    { id: 6, lockerNumber: 6, status: "Empty" },
+    { id: 7, lockerNumber: 7, status: "Empty" },
+  ]);
+
+  // Mock function to assign package
+  const assignPackage = () => {
+    if (!apartmentNumber || !selectedLocker) {
+      return;
+    }
+    // Mock implementation
+  };
+
+  // Mock function to discard package
+  const discardPackage = (id: number) => {
+    // Mock implementation
+  };
+
+  return (
+    <main className="min-h-screen relative pb-16 md:pb-0">
```

Comment:
I would divide this into two components. : the assigning component and the overview component. I am not sure what to call the components.